lib/gems/pending/appliance_console/database_replication.rb

```diff
@@ -52,6 +53,10 @@ def create_config_file(host)
         f.puts("conninfo='host=#{host} user=#{database_user} dbname=#{database_name}'")
         f.puts("use_replication_slots=1")
         f.puts("pg_basebackup_options='--xlog-method=stream'")
+        f.puts("failover=automatic\n")
```

Comment:
Should the content generation be moved into a method then just `File.write` that?  Then you can just test that the method generates the correct content and not worry about a test that writes a file.